modules/dnn/test/test_halide_layers.cpp

```diff
@@ -216,7 +212,7 @@ INSTANTIATE_TEST_CASE_P(Layer_Test_Halide, Deconvolution, Combine(
 /*in size*/  Values(Size(5, 6)),
 /*kernel*/   Values(Size(3, 1), Size(1, 3)),
 /*pad*/      Values(Size(1, 0), Size(0, 1)),
-/*dilation*/ Values(Size(1, 1), Size(2, 2)),
+/*dilation*/ Values(Size(1, 1)),
```

Comment:
Why is this test case reduced?

Reply:
@alalek, I found that it's not implemented even in OpenCV backend:

there is dilation for convolution layer:
https://github.com/opencv/opencv/blob/db900af8c7a849539aff3941c2a788c190bb317b/modules/dnn/src/layers/convolution_layer.cpp#L287-L295

but not for deconvolution one:
https://github.com/opencv/opencv/blob/db900af8c7a849539aff3941c2a788c190bb317b/modules/dnn/src/layers/convolution_layer.cpp#L1287-L1301